cannot find package "golang.org/x/crypto/acme/autocert"

When building the Docker image from 83836bc45dce88f6a0474f8b0853419ade787af2, I get:

```
$ docker build -t my/ssl-proxy .
[...]
Step 6/6 : RUN make
 ---> Running in 0cc1b0b6fa3d
go mod download
warning: pattern "all" matched no module dependencies
go build -o ssl-proxy
main.go:16:2: cannot find package "golang.org/x/crypto/acme/autocert" in any of:
        /usr/local/go/src/golang.org/x/crypto/acme/autocert (from $GOROOT)
        /go/src/golang.org/x/crypto/acme/autocert (from $GOPATH)
make: *** [Makefile:6: build] Error 1
The command '/bin/sh -c make' returned a non-zero code: 2
```

This is on Ubuntu 18.04.4, using Docker version 18.09.7, build 2d0083d (from the Ubuntu repository).

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine3.9
+FROM golang:1.13.7-alpine3.11
 WORKDIR /go/src/github.com/suyashkumar/ssl-proxy
 RUN apk add --no-cache make git zip
 RUN go get -u github.com/golang/dep/cmd/dep

